src/Schema/AbstractSchema.php

```diff
@@ -261,7 +262,7 @@ public function getTableForeignKeys(string $name, bool $refresh = false): array
      */
     public function getTableIndexes(string $name, bool $refresh = false): array
     {
-        /** @psalm-var mixed $tableIndexes */
+        /** @var IndexConstraint[]|null $tableIndexes */
```

Comment:
Are there cases when it returns `null`?

Reply:
Hm... I don't know. If no, then we can remove check "is_array".

Reply:
> Hm... I don't know. If no, then we can remove check "is_array".

Yes, looks like this.

Reply:
Done